docs/policies/alcf-acknowledgement-policy.md

```diff
@@ -8,11 +8,17 @@ To publish technical reports and research papers using the ALCF AI testbeds, we
 
 For guidance on acknowledgements, please see the following sample policies:
 
-## ALCF Only Acknowledgement
-This research used resources of the Argonne Leadership Computing Facility, which is a DOE Office of Science User Facility supported under Contract DE-AC02-06CH11357.
+## ALCF Only Acknowledgement. 
```

Comment:
should this period be here @jini-ramprakash ?

Reply:
You are right - that shouldn't be there... let me remove and re-submit.